yb-voyager/src/srcdb/postgres.go

```diff
@@ -1502,3 +1509,26 @@ func (pg *PostgreSQL) listSchemasMissingUsagePermission() ([]string, error) {
 
 	return schemasMissingUsagePermission, nil
 }
+
+func (pg *PostgreSQL) CheckDependencies() error {
+	var errs []string
+
+	// List of pg binaries to check
+	binaries := []string{"pg_dump", "pg_restore"}
```

Comment:
Lets have a global variable for this with name: `pgExportDependecies` or something

Reply:
Done. But the location of the entire function has been changed. I have added it to export.go. So please have a look there.